networking/multiple_networks/configuring-additional-network.adoc

```diff
@@ -92,13 +92,15 @@ spec:
     type: Raw
 ----
 <1> An array of one or more additional network configurations.
-
 <2> The name for the additional network attachment that you are
 creating. The name must be unique within the specified `namespace`.
-
 <3> The namespace to create the network attachment in. If
-you do not specify a value, then the `default` namespace is used.
-
+you do not specify a value then the `default` namespace is used.
++
+[IMPORTANT]
+====
+To prevent namespace issues for the OVN-Kubernetes network plugin, do not name your additional network attachment as "default", because this namespace is reserved for the `default` additional network attachment.
```

Comment:
User input is also in backticks.
```suggestion
To prevent namespace issues for the OVN-Kubernetes network plugin, do not name your additional network attachment `default`, because this namespace is reserved for the `default` additional network attachment.
```

Reply:
Just getting back to this PR and it seems like I happened to add a merge review in progress label right before the bot removed the merge review needed label, meaning I caught it mistakenly for the split second it was in the queue. 

I'll continue reviewing this PR anyways, but please also address Jeana's feedback in the meantime

Reply:
:heavy_check_mark: